frontend/src/hooks/useMessageValidation.ts

```diff
@@ -0,0 +1,38 @@
+import { useFormik } from "formik";
+import * as Yup from "yup";
+
+const regExp =
+    /^(https?):\/\/(?:www\.)?[a-zA-Z]{2,}\.[a-z]{2,5}(?!.*(\/{2}|https?|:|\.| )).*$/;
+export const maxCount = 500;
+
+interface IMessageVal {
+    handleSubmitForm: (content: string) => void;
+    defaultContent?: string;
```

Comment:
Use here "content" this prefix is unnecessary, u compicate code,

like here:
```ts
const formik = useMessageVal({ handleSubmitForm, defaultContent: content });
```

u can't send "content" u must assign it to "defaultContent"

Reply:
done